lib/queries/index.js

```diff
@@ -0,0 +1,14 @@
+import getUserListById from './getUserListById.gql';
+import removeMedia from './removeMedia.gql';
+import searchAnime from './searchAnime.gql';
+import updateMedia from './updateMedia.gql';
+
+const ops = { getUserListById, removeMedia, searchAnime, updateMedia };
+
+// transform each ops as a function that requests query
+Object.keys(ops).forEach(key => {
```

Comment:
Cant we do this with reduce as well?

Reply:
That would mean system would have to internally create an extra array, right?

Reply:
True, btw do u want me to press merge or u wanna merge

Reply:
I can't merge

Reply:
gg lol will do that